src/screens/FeedScreen/elements/Search.js

```diff
@@ -47,18 +47,12 @@ const Search = ({
 };
 
 export const HeaderStyles = StyleSheet.create({
-  container: {
-    flexDirection: 'row',
-    backgroundColor: COLORS.white,
-    marginBottom: SIZES.base,
-    marginHorizontal: SIZES.base
-  },
   searchContainer: {
     flex: 1
   },
   wrapperSearch: {
     flex: 1,
-    backgroundColor: COLORS.lightgrey,
+    backgroundColor: '#F5F5F5',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the `backgroundColor` property from `COLORS.lightgrey` to `'#F5F5F5'` might affect the consistency of the application's color scheme. It is generally a good practice to use a centralized color system (like the `COLORS` object you had before) instead of hardcoding color values. This makes it easier to maintain and update the color scheme of the entire application.

```diff
-    backgroundColor: '#F5F5F5',
+    backgroundColor: colors.lightGrey,
```

Please ensure that `colors.lightGrey` is equivalent to `'#F5F5F5'` in your new color system. If not, adjust accordingly.


<!-- This is an auto-generated comment by OSS CodeRabbit -->